### PR TITLE
Sonar @SuppressWarnings("java:S115")

### PR DESCRIPTION
**Pull request by jepsar**
@melloware I would like to get rid of SonarQube issues with high severity. What's your opinion on suppressing issues that we cannot fix?

### CODE REVIEW

**Comment by melloware** [comment]
I have been doing that supressing things like where it doesn't like PropertyKeys enumeration is in UPPERCASE etc.  So feel free!

File: core/src/main/java/org/primefaces/extensions/component/calculator/Calculator.java [hunk 1]
```diff
@@ -60,6 +60,7 @@
 
     private Locale appropriateLocale;
 
+    @SuppressWarnings("java:S115")
```
Comment (melloware) [inline core/src/main/java/org/primefaces/extensions/component/calculator/Calculator.java:63]: Oh cool, we probably need this supresswarning on every single PropertyKeys.

**Comment by jepsar** [comment]
@melloware there are a few complexity issues which I didn't want to solve here. I think we need PRs per class. Especially the exporter is horrible.

**Comment by melloware** [comment]
Agrees the Exporter is in really bad shape. I cleaned it up a few years ago but only really glaring things. I originally wanted to delete the component since PF version has almost the same features now but I think people still use this one. Good luck!

**Review by melloware — APPROVED** [review]

**Pull request merged**